src/index.html

```diff
@@ -1,13 +1,66 @@
 <!doctype html>
-<html lang="en">
+<html class="card" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="card__body">
+    <div data-qa="card" class="card__content">
+      <img
+        class="card-image"
```

Comment:
```suggestion
        class="card__image"
```

Reply:
https://github.com/mate-academy/layout_product-cards/pull/3617/files#r1324335428
fix all you class naming to bem methodology
https://en.bem.info/methodology/